release/release_tests.yaml

```diff
@@ -5791,9 +5791,9 @@
     cluster_compute: dataset/multi_node_train_4_workers.yaml
 
   run:
-    timeout: 3600
+    timeout: 21600
     prepare: python setup_chaos.py --kill-workers --kill-interval 50 --max-to-kill 20 --task-names "ReadImage->Map(crop_and_flip_image)"
-    script: python dataset/multi_node_train_benchmark.py --num-workers 4 --file-type image
+    script: python dataset/multi_node_train_benchmark.py --num-workers 4 --file-type image --use-gpu --num-epochs 3
```

Comment:
Thanks! Could you also add the extra map to the task name so that the arg is `"ReadImage->Map(wnid_to_index)->Map(crop_and_flip_image)"`? Also now that it runs longer maybe we should spread the kills more evenly `--kill-interval 200 --max-to-kill 50`